src/api/reactivity-advanced.md

```diff
@@ -33,8 +33,8 @@ Versión poco profunda de [`ref()`](./reactivity-core.html#ref).
   ```
 
 - **Véase también**
-  - [Guía - Reducción de la Sobrecarga de Reactividad en Estructuras Inmutables de Gran Tamaño](/guide/best-practices/performance.html#reduccion-de-la-sobrecarga-de-reactividad-en-estructuras-inmutables-de-gran-tamano)
-  - [Guía - Integración con los Sistemas de Estado Externos](/guide/extras/reactivity-in-depth.html#integracion-con-los-sistemas-de-estado-externos)
+  - [Guía - Reducción de la Sobrecarga de Reactividad en Estructuras Inmutables de Gran Tamaño](/guide/best-practices/performance#reduccion-de-la-sobrecarga-de-reactividad-en-estructuras-inmutables-de-gran-tamano)
+  - [Guía - Integración con los Sistemas de Estado Externos](/guide/extras/reactivity-in-depth#integracion-con-los-sistemas-de-estado-externos)
```

Comment:
change to #reduce-reactivity-overhead-for-large-immutable-structures and #integration-with-external-state-systems respectively